django_php_bridge/backends/db.py

```diff
@@ -1,20 +1,56 @@
 import phpserialize
-
 from django.contrib.sessions.backends.db import SessionStore as DbStore
 
+
 class SessionStore(DbStore):
-    '''
+  '''
     Thanks to Alex Ezell for assistance on this php-django session engine.
     http://groups.google.com/group/django-users/browse_thread/thread/f5b464379f2e4154/e358161c95e507c0
 
     Override the default database session backend to use php-style serialization.
     '''
-    def __init__(self, session_key=None):
-        # call the super class's init
-        super(SessionStore, self).__init__(session_key)
+  def __init__(self, session_key=None):
+    # call the super class's init
+    super(SessionStore, self).__init__(session_key)
+
+  def _wrap_type(self, obj):
+    out = obj
+    if isinstance(obj, (list, tuple, dict)):
+      out = {}
+      if isinstance(obj, dict):
+        iterable = obj.items()
+        for key, value in iterable:
+          out[key] = self._wrap_type(value)
+      else:
+        out["_bridge_type"] = type(obj).__name__
```

Comment:
This is where most of the magic happens, every object is wrapped up and expects to know it's pythonic type as well as value. I'd be interested to see how the author proposes implementing this in PHP.

It seems that something like redis with it's own integral hash type would be a better fit than MySQL. I Realise this is all old code, but forcing python to use php serialized objects is not going to help you much and in-fact makes the whole process of migration harder